best_practices.rst

```diff
@@ -0,0 +1,307 @@
+The Symfony Framework Best Practices
+====================================
+
+The Symfony Framework is well-known for being *really* flexible and is used to
+build from micro-sites to enterprise applications. This guide describes the
+**best practices for developing web applications with Symfony** that fit the
+philosophy envisioned by the original Symfony creators. You should only read
+this guide if you already have experience developing Symfony applications.
```

Comment:
why? a new comers can get some good hint and tips reading IMO

Reply:
Not sure about this. For example, we mention in the "best practices" that *"you should not use bundles to organize your own code"*. Cool but ... what's a "bundle"?

"Best Practices"is just a listing of things you should do or not do in a Symfony app (according to its creators) ... so, you need to know Symfony first.